WordPress/Classes/Utility/Editor/EditorFactory.swift

```diff
@@ -12,16 +12,16 @@ class EditorFactory {
 
     // MARK: - Editor: Instantiation
 
-    func instantiateEditor(for post: AbstractPost, replaceEditor: @escaping ReplaceEditorBlock) -> EditorViewController {
+    func instantiateEditor(for post: AbstractPost, shouldLoadAutosave: Bool = false, replaceEditor: @escaping ReplaceEditorBlock) -> EditorViewController {
```

Comment:
That's a good point, I hadn't noticed the inconsistency. 
I'm going to propose `loadAutosaveRevision` because I think it's best for the argument name to read as an assertion, and it also keeps things short and sweet.